tasks/main.yml

```diff
@@ -2,6 +2,8 @@
 - name: Hard set the hostname
   hostname: name={{ ansible_fqdn }}
 
+# - shell: hostname {{ ansible_fqdn }}
```

Comment:
Sure. Give me a few minutes while I figure out how. I am painfully new to
git pull requests still.

On Mon, Nov 23, 2015 at 4:33 PM, Greg Swift <notifications@github.com>
wrote:

> In tasks/main.yml
> <https://github.com/gregswift/ansible-ipaserver/pull/1#discussion_r45671176>
> :
>
> > @@ -2,6 +2,8 @@
> >  - name: Hard set the hostname
> >    hostname: name={{ ansible_fqdn }}
> >
> > +# - shell: hostname {{ ansible_fqdn }}
>
> Can you remove this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gregswift/ansible-ipaserver/pull/1/files#r45671176>.
>



-- 


Joanna Delaporte
Linux Systems Administrator | Parkland College
joannadelaporte@gmail.com